packages/js-runtime/src/__tests__/fetch.test.ts

```diff
@@ -122,20 +122,20 @@ describe('Headers', () => {
     it('should set without init', () => {
       const headers = new Headers();
       headers.set('a', 'b');
-      headers.set('c', 'd');
+      headers.set('C', 'd');
       expect(headers.get('a')).toEqual('b');
       expect(headers.get('c')).toEqual('d');
     });
 
     it('should set with init', () => {
       const headers = new Headers({
         a: 'b',
-        c: 'd',
+        C: 'd',
       });
       headers.set('a', 'e');
       headers.set('c', 'f');
       expect(headers.get('a')).toEqual('e');
-      expect(headers.get('c')).toEqual('f');
+      expect(headers.get('C')).toEqual('f');
     });
```

Comment:
Could you add another test for `set()`, by setting a new header with the key in uppercase and verifying if the key has been converted to lowercase?

Reply:
Done.